src/main/java/ai/deepcode/javaclient/DeepCodeRestApi.java

```diff
@@ -336,66 +296,11 @@ public static CreateBundleResponse extendBundle(
     return result;
   }
 
-  private interface UploadFilesCall {
-    @retrofit2.http.Headers("Content-Type: application/json;charset=utf-8")
-    @POST("file/{bundleId}")
-    Call<Void> doUploadFiles(
-        @Header("Session-Token") String token,
-        @Path(value = "bundleId", encoded = true) String bundleId,
-        @Body List<FileHash2ContentRequest> listHash2Content);
-  }
-
-  /**
-   * Uploads missing files to a bundle.
-   *
-   * @param token
-   * @param bundleId
-   * @param request List&lt;FileHash2ContentRequest&gt;
-   * @return EmptyResponse with return code and description.
-   */
-  public static EmptyResponse UploadFiles(
-      String token, String bundleId, List<FileHash2ContentRequest> request) {
-    UploadFilesCall uploadFilesCall = retrofit.create(UploadFilesCall.class);
-    Response<Void> retrofitResponse;
-    try {
-      retrofitResponse = uploadFilesCall.doUploadFiles(token, bundleId, request).execute();
```

Comment:
Are we completely ignoring missing files response now? Wondering what's the performance implication here. cc @ArtsiomCh

Reply:
Didn't dive deep into the code yet. But from the log output I can see we honoring `missingFiles` in the right way. 
But may be I miss some edge cases...

Reply:
We don't ignore it. We just use a different endpoint (extendBundle). uploadFiles does not exist anymore on the API site. And missingFiles is existent and a thing there and in checkBundle :).